### ISSUE TITLE
Consider replacing file dialogs

### ISSUE THREAD

**Issue by nickzoic** [CountESS-Project/CountESS#36]
[Tk's built-in file dialogs](https://tcl.tk/man/tcl8.6/TkCmd/getOpenFile.htm#M16) look pretty awful and see also #26 #27

Consider replacing with a custom file selection widget.

**Comment by afrubin** [comment]
I agree with this but it seems very much like a "nice to have" rather than a key issue. Are there other open source file selection widgets for Tk that are already built that we can easily pull in?
